src/runtime/Makefile

```diff
@@ -347,7 +347,7 @@ ifneq (,$(QEMUCMD))
     CONFIG_PATHS += $(CONFIG_PATH_QEMU_SEV)
 
     SYSCONFIG_QEMU_SEV = $(abspath $(SYSCONFDIR)/$(CONFIG_FILE_QEMU_SEV))
-    SYSCONFIG_PATHS += $(SYSCONFIG_QEMU_SEV)
+	SYSCONFIG_PATHS_SEV += $(SYSCONFIG_QEMU_SEV)
```

Comment:
Please, do not change the indentation here.

Reply:
I've pushed the fixup so that we can merge this